portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -53,21 +57,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String requestData = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
-    String commentString = requestData.replaceAll("(\r\n|\n)", "<br />");
-    Entity commentEntity = Comment.createCommentEntity(commentString);
+    String[] requestData = request.getReader().lines().collect(Collectors.joining(System.lineSeparator())).split("\\n+");
```

Comment:
Same comment as above about a.b().c().d().e().f(x.y(z)).g(regexp) this is painful to parse.

Also please make your lines 80chars or less.  You can use a Java linter to enforce this and also modify your IDE.